x-pack/plugins/fleet/server/services/agent_policies/full_agent_policy.ts

```diff
@@ -202,7 +202,10 @@ export async function getFullAgentPolicy(
     NonNullable<FullAgentPolicy['output_permissions']>
   >((outputPermissions, outputId) => {
     const output = fullAgentPolicy.outputs[outputId];
-    if (output && output.type === outputType.Elasticsearch) {
+    if (
+      output &&
+      (output.type === outputType.Elasticsearch || output.type === outputType.RemoteElasticsearch)
```

Comment:
I found `output_permissions` are needed for remote-elasticsearch output, though it was not mentioned in the definition.